draft-ietf-core-cf-reg-update.md

```diff
@@ -193,30 +196,33 @@ The 256-9999 range now has registration procedures requiring IETF Review with Ex
 
 The 10000-64999 range now has two separate registration procedures.
 If the registration consists solely of a registered media type name in the "Content Type" field, without any parameter names or "Content Coding", and the media type has not yet been used in this registry, then the policy is FCFS, as before.
-In all other cases, the policy will be Expert Review, following the checklist described in {{checks}}.
+In all other cases, the policy is "Expert Review," following the procedure described in {{checks}}.
 
 A new column with the title "Note" has been added to the registry, which contains information about the expected review procedure.
 
 ## Temporary Content-Format Registrations
 
-This section clarifies that the CoAP Content-Formats registry allows temporary registrations within the 0-255 and 256-9999 ranges.
+This section clarifies that the "CoAP Content-Formats" registry allows temporary registrations within the 0-255 and 256-9999 ranges.
 The range 10000-64999 does not allow temporary registrations.
 
-A temporary registration may be created for example by an IANA early allocation action, requested by the authors of an Internet Draft in the IETF stream.
-Or it may be created because the referenced media type is still provisional (that is, included in the IANA Provisional Standard Media Type Registry).
+A temporary registration may be created for example by an IANA early allocation action, as requested by the authors of an Internet-Draft in the IETF stream.
+Alternatively, it may be created because the referenced media type is still provisional (that is, included in the IANA "Provisional Standard Media Type" registry {{IANA.provisional-standard-media-types}}).
 
-A temporary registration is marked by an IANA note with the label "TEMPORARY" in the corresponding registry entry.
-Once the required review procedure for the temporary ID has successfully completed, and the referenced media type is included in the IANA Media Types registry, IANA must remove the "TEMPORARY" label so that the entry becomes permanent.
+A temporary registration is marked by an IANA note with the label "TEMPORARY" in the  "Content Type" column of the corresponding registry entry.
```

Comment:
I think here naming the specific column is too detailed. IANA will mark it somewhere on the line, and that might change over time (e.g. do the label in the Reference column if they like that more).

Reply:
@EskoDijk what about:

> _A temporary registration is marked by IANA with the label "TEMPORARY" in the corresponding registry entry._

Reply:
done in [3d11d2b](https://github.com/core-wg/cf-reg-update/pull/40/commits/3d11d2b7bcd22c3a978a96225772eae8a1544067)